distro/run/pom.xml

```diff
@@ -16,10 +16,9 @@
   <packaging>pom</packaging>
 
   <properties>
-    <!-- Version overrides for logback and log4j should be removed 
+    <!-- Version override for log4j should be removed 
          after bumping the Spring Boot version -->
     <version.log4j2>2.17.1</version.log4j2>
-    <version.logback>1.2.9</version.logback>
```

Comment:
If I am not mistaken, you can remove both overrides here, as Spring Boot 2.6.3 updates to logback 1.2.10 and log4j 2.17.1. Don't forget to remove both from `dependencyManagement` as well then.